core/templates/domain/state_card/state-card.model.ts

```diff
@@ -22,7 +22,7 @@ import cloneDeep from 'lodash/cloneDeep';
 import { AppConstants } from 'app.constants';
 import { AudioTranslationLanguageService } from
   'pages/exploration-player-page/services/audio-translation-language.service';
-import { Interaction } from 'domain/exploration/InteractionObjectFactory';
+import { Interaction, InteractionSpecsKey } from 'domain/exploration/InteractionObjectFactory';
```

Comment:
@mridul-netizen Didn't you also define this somewhere (`InteractionSpecsKey`)? We should probably think what is the ideal place for it.

Reply:
Thanks! That makes sense!